includes/fields/class-field.php

```diff
@@ -122,6 +122,13 @@ abstract class Field {
 	 */
 	protected $attributes = [];
 
+	/**
```

Comment:
Please avoid making the API more complex at all costs. We already have too many undocumented parameters, please always let me know in tg if you're considering to add a new helper, hook, flag, etc.

In this case the easiest solution is filtering the {model}_filter form in a loop (using Attribute models, since the search is implemented by Attribute component please add this logic there), and appending the link HTML to the form footer, with data-attribute that can be used for JS logic. Other changes are not required.